marimo/_runtime/runner/hooks_post_execution.py

```diff
@@ -339,6 +340,26 @@ def format_output() -> formatting.FormattedOutput:
         )
 
 
+@kernel_tracer.start_as_current_span("run_pytest")
+def _attempt_pytest(
+    cell: CellImpl,
+    runner: cell_runner.Runner,
+    run_result: cell_runner.RunResult,
+) -> None:
+    del run_result
+    if cell._test:
```

Comment:
Only run if the runner isn't lazy?

Reply:
Implemented as a conditional post-execution hook. With the experimental flag is in place, this would be a no-op with lazy

But definitely for removing the flag. I was thinking maybe a separate run button?

![1740746561216496456820358512708](https://github.com/user-attachments/assets/9d1fd924-60f8-43d7-a9c1-81836cfa25bf)